### PR TITLE
Replace animate.css with keyframes and classes

### PR DESCRIPTION
**Pull request by thundron**
Fixes: #1054
Change-type: patch
Signed-off-by: Lorenzo Alberto Maria Ambrosi <lorenzothunder.ambrosi@gmail.com>

Connects-to: # 
See: <url> 
Depends-on: <url> 
Change-type: major|minor|patch 

---
##### Contributor checklist

- [x] I have rebuilt the README with `npm run build:docs`
- [ ] I have regenerated screenshots for any affected components with `npm run generate-screenshots`
- [ ] I have regenerated jest snapshots for any affected components with `npm run jest -- -u`

##### Reviewer Guidelines
- When submitting a review, please pick:
  - '*Approve*' if this change would be acceptable in the codebase (even if there are minor or cosmetic tweaks that could be improved).
  - '*Request Changes*' if this change would not be acceptable in our codebase (e.g. bugs, changes that will make development harder in future, security/performance issues, etc).
  - '*Comment*' if you don't feel you have enough information to decide either way (e.g. if you have major questions, or you don't understand the context of the change sufficiently to fully review yourself, but want to make a comment)
---

### CODE REVIEW

File: src/animations.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { css } from 'styled-components';
+
+export const animations = css`
```
Comment (sradevski) [inline src/animations.ts]: You can also use `keyframes`, I believe (same below): https://styled-components.com/docs/basics#animations
```suggestion
export const animations = keyframes`
```

**Comment by thundron** [comment]
@balena-ci retest

File: src/components/Notifications/index.tsx [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
 import ReactNotification, { store } from 'react-notifications-component';
-import 'react-notifications-component/dist/theme.css';
```
Comment (sradevski) [inline src/components/Notifications/index.tsx]: Are the styles below everything that is in the theme?

Comment (thundron) [reply src/components/Notifications/index.tsx]: No they're just for the storybook as I was doing some tests on Etcher too. WRT porting them all: should I make a file inside the Notification component with the theme? Feels better than a `styles.ts` in the src folder or writing them inside index.ts of the component, which would make it quite uncomfortable to read

Comment (thundron) [reply src/components/Notifications/index.tsx]: No, I was just testing stuff in the storybook + Etcher, including them in a separate commit for semplicity

**Comment by sradevski** [comment]
@thundron Let's add https://github.com/euheny/babel-plugin-transform-css-import-require-to-string instead of copy-pasting CSS. We actually have https://github.com/balena-io-modules/rendition/blob/master/scripts/tsifyCss.js to do similar things for the terminal and the MD editor, but we can remove that script as well and use the babel transform instead.

**Comment by thundron** [comment]
Sure, on it

**Pull request closed without merging**